src/pages/overview/pdf-services-api/howtos/webhook-notification.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Webhook Notification support for Adobe PDF Services APIs | Adobe PDF Services
+---
+# **Webhook Notification support for Adobe PDF Services APIs**
+
+Adobe PDF Services APIs have added support for webhook notification. After invoking the PDF Service API, the client need not required to continuously poll for the result of the job and they will get automatically notifies about job completion along with the status.
+
+The client needs to add an extra parameter in the input request called **notifier**. It is an array of webhook notification related information and each element contains following parameters:
+
+1. **type**: Type of notifier needs to be invoked. Currently supported notifier type is [CALLBACK](#callback-notifier).
+2. **data**: Information related to webhook notification data. It depends on type of notifier being used.
+
+## Sample Request Body
+```json
+{
+  "assetID": "{{assetID}}",
+  ....,
+  ///REST API Operation related details
+  ....
+  
+  "notifier": [
+    {
+      "type": "<NOTIFIER_TYPE>",
+      "data": {
+          //<NOTIFICATION_DATA>
+        }
+      }
+    }
+  ]
+}
+```
+
+<InlineAlert slots="text"/>
+<div>
+Webhook notification support is only available in REST APIs. It is currently <b>not</b> supported in SDKs.
+</div>
+
+## **CALLBACK Notifier**
+Users needs to create a HTTPS POST URL, and share that URL and related headers information in the request. On the completion of job, they will get automatically notified about the status of that job on that URL.
+
+As the name suggests, the notification *type* will be **CALLBACK**.
+
+### Parameters for notification **data**
+
+1. **url** _(Required)_: HTTPS POST URL created by the user.
+2. **headers** _(Optional)_: It is map of key value pair and represent headers related information required to call the URL.
+
+### Sample Request Body
+
+```json
+{
+  "assetID": "{{assetID}}",
+  ....,
+  ///REST API Operation related details
+  ....
+  
+  "notifier": [
+    {
+      "type": "CALLBACK",
+      "data": {
+        "url": "https://dummy.callback.org/",
+        "headers": {
+          "x-api-key": "dummykey",
+          "access-token": "dummytoken"
+        }
+      }
+    }
+  ]
+}
+```
+
+### Notification Payload 
+The notification payload sent to CALLBACK URL is same as that of **GET STATUS** response.
+
+#### Successful Scenario
+In case of successful job completion, following **sample payload** will be sent to CALLBACK URL:
```

Comment:
In case of successful job completion, the following **callback payload** will be sent to callback url: